clusters/talos-robbinsdale/apps/immich/pvc.yaml

```diff
@@ -9,3 +9,5 @@ spec:
     requests:
       storage: 200Gi
   storageClassName: nfs-truenas
```

Comment:
@rajsinghtech do you think it would be better to move this to Ceph NVME? Would Immich/pics benefit from that? Long term might grow but rn <100GB.